assets/src/edit-story/components/helpCenter/menu/footer.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { themeHelpers } from '../../../../design-system';
+
+const Panel = styled.div`
+  padding: 16px 0 24px 0;
+  border-top: 1px solid ${({ theme }) => theme.colors.bg.tertiary};
+`;
```

Comment:
Since panel repeats, is there any use in extracting to make it easier to share?

Reply:
I just named em the same thing cus they all play a similar function, but they all have different paddings and borders. I could make it a component that takes padding and border props, but kinda feels like an unnecessary layer on top of styled components. I think it would just result in more code to do the same thing